ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -367,6 +367,23 @@ def logical_or(self, y, out=None, name=None):
     def bitwise_xor(self, y, out=None, name=None):
         return paddle_frontend.bitwise_xor(self, y)
 
+    @with_supported_dtypes(
+        {
+            "2.5.1 and below": (
+                "bool",
+                "int8",
+                "int16",
+                "int32",
+                "int64",
+                "float32",
+                "float64",
+            )
+        },
+        "paddle",
+    )
+    def logical_xor(self, y, out=None, name=None):
```

Comment:
why have you implemented `logical_xor` when the PR is about `broadcast_tensors` ?